src/ip.cpp

```diff
@@ -596,9 +601,11 @@ int zmq::make_fdpair (fd_t *r_, fd_t *w_)
     rc = bind (listener, const_cast<sockaddr *> (address.addr ()),
                address.addrlen ());
     if (rc != 0) {
-        errno = wsa_error_to_errno (WSAGetLastError ());
```

Comment:
why is this being removed?

Reply:
It's unused, so I figured it wasn't useful to keep it. I put it back, though.